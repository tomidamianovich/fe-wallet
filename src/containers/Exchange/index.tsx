import React from 'react';
import MaintenancePage from "../../components/MaintenancePage";
import { withTranslation, WithTranslation } from 'react-i18next';
import './styles/index.scss';

type Props = {} & WithTranslation;

const Exchange: React.FC<Props> = ({ t }) => 
  <div className="fe-wallet__exchange">
    <MaintenancePage pageName={t('pages.exchange')} />
  </div>
;

export default withTranslation()(Exchange);
