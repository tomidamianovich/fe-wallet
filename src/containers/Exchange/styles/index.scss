.fe-wallet__exchange {
  display: flex;
  width: 100%;
}
