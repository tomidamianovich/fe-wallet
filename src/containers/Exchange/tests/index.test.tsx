import { render } from '@testing-library/react';
import Exchange from '../';

test('renders Exchange main page', () => {
  const { asFragment } = render(<Exchange />);
  expect(asFragment()).toMatchSnapshot();
});