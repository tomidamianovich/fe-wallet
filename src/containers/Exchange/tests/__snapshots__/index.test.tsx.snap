// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Exchange main page 1`] = `
<DocumentFragment>
  <div
    class="fe-wallet__exchange"
  >
    Exchange
  </div>
</DocumentFragment>
`;
